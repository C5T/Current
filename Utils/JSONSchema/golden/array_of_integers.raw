{"Array":{"element":{"Integer":{"sum":6,"sum_squares":14.0,"can_be_unsigned":true,"can_be_microseconds":false,"instances":3,"nulls":0}},"instances":1,"nulls":0}}