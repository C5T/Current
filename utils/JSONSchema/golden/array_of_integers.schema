// Autogenerated schema inferred from input JSON data.

using Schema = std::vector<uint64_t>;
