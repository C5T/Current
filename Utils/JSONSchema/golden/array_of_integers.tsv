Field	Type	Set	Unset/Null	Values	Details
Schema	Array	1	0
Schema[]	Integer	3	0	Mean 2, StdDev 0.816497
